SingularityUI/app/views/tail.coffee

```diff
@@ -1,5 +1,7 @@
 View = require './view'
 
+TERMINAL_TASK_STATES = ['TASK_KILLED', 'TASK_LOST', 'TASK_FAILED', 'TASK_FINISHED']
```

Comment:
we have this floating around in a few places -- we should consolidate them in a constants file